sdk/core/core-http/src/serviceClient.ts

```diff
@@ -184,6 +184,7 @@ export class ServiceClient {
    */
   constructor(
     credentials?: TokenCredential | ServiceClientCredentials,
+    /* eslint-disable-next-line @azure/azure-sdk/ts-naming-options */
     options?: ServiceClientOptions
```

Comment:
@willmtemple hi! This TS rule doesn't work well with constructor options :(

Reply:
Most of our constructors have an `options` parameter. I'll have to dig in to why this was flagged.